morpheus.code-workspace

```diff
@@ -1,623 +1,618 @@
-						"text": "-interpreter-exec console \"skip -gfi **/bits/*.h\""
-					},
-				],
-				"symbolLoadInfo": {
-					"loadAll": false,
-					"exceptionList": "libmrc*.so;cudf_helpers.*;executor.*;morpheus.*;node.*;options.*;pipeline.*;segment.*;subscriber.*;stages.*;messages.*;common*.so"
-				},
-				"miDebuggerPath": "gdb",
-				"sourceFileMap": {
-					"${workspaceFolder}": {
-						"editorPath": "${workspaceFolder}",
-						"useForBreakpoints": "true"
-					}
-				},
-			},
-			{
-				"name": "Debug MRC from Python (Morpheus-AE)",
-				"type": "cppdbg",
-				"request": "launch",
-				"program": "python",
-				"args": [
-					"./morpheus/cli.py",
-					"--log_level=DEBUG",
-					"run",
-					"--num_threads=1",
-					"--pipeline_batch_size=128",
-					"--model_max_batch_size=128",
-					"--use_cpp=False",
-					"pipeline-ae",
-					// "--ae_path=models/hammah-models/hammah-role-g-20211017.pkl",
-					"--ae_path=../data/ae_model.pkl",
-					"from-cloudtrail",
-					"--input_glob=models/datasets/validation-data/dfp-cloudtrail-role-g-validation-data.csv",
-					// "--input_glob=./data/red_team.csv",
-					"--max_files=200",
-					"--iterative",
-					"deserialize",
-					"preprocess",
-					"inf-triton",
-					"--model_name=autoencoder-onnx",
-					"--server_url=localhost:8001",
-					"timeseries",
-					"--resolution=10m",
-					"monitor",
-					"--description",
-					"Inference Rate",
-					"--smoothing=0.001",
-					"--unit",
-					"inf",
-					// "add-class",
-					// "filter",
-					// "serialize",
-					// "--include",
-					// "timestamp",
-					// "--exclude",
-					// "^_ts_",
-					// "--exclude",
-					// "^nvidia_smi_log",
-					// "to-kafka",
-					// "--output_topic",
-					// "inference_output",
-					// "to-file",
-					// "--filename=./.tmp/detections.json",
-					// "--overwrite",
-				],
-				"stopAtEntry": false,
-				"cwd": "${workspaceFolder}",
-				"environment": [
-					{
-						"name": "MORPHEUS_ROOT",
-						"value": "${workspaceFolder}"
-					},
-					{
-						"name": "GLOG_v",
-						"value": "10"
-					},
-					{
-						"name": "CUDA_LAUNCH_BLOCKING",
-						"value": "1"
-					}
-				],
-				"externalConsole": false,
-				"MIMode": "gdb",
-				"setupCommands": [
-					{
-						"description": "Enable pretty-printing for gdb",
-						"text": "-enable-pretty-printing",
-						"ignoreFailures": true
-					},
-					{
-						"description": "Skip stdio-common files",
-						"text": "-interpreter-exec console \"skip -gfi **/bits/*.h\""
-					},
-				],
-				"symbolLoadInfo": {
-					"loadAll": false,
-					"exceptionList": "libmrc*.so;cudf_helpers.*;executor.*;morpheus.*;node.*;options.*;pipeline.*;segment.*;subscriber.*;stages.*;messages.*;common*.so"
-				},
-				"miDebuggerPath": "gdb",
-				"sourceFileMap": {
-					"${workspaceFolder}": {
-						"editorPath": "${workspaceFolder}",
-						"useForBreakpoints": "true"
-					}
-				},
-			},
-		],
-		"compounds": []
-	}
+    "folders": [
```

Comment:
Are all of these changes intentional?

Reply:
Yes, just sorted the JSON to make diffs easier in the future.